scripts/build_mkl_gpu.sh

```diff
@@ -33,7 +33,8 @@ export CUDA_TOOLKIT_ROOT_DIR=""
 export CMAKE_Fortran_FLAGS="-fiopenmp -fopenmp-targets=spir64 -DINTEL_SDK"
 export CMAKE_C_FLAGS="-O3 -fiopenmp -fopenmp-targets=spir64 -D__STRICT_ANSI__ -DINTEL_SDK"
 
-export EXTRA_LINK_FLAGS="-L/soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/mkl/2021u3_20210525/lib/intel64 -lmkl_lapack95_lp64 -lmkl_intel_lp64 -lmkl_core -lmkl_intel_thread -lpthread -lm -lmkl_intel_ilp64  -lmkl_sycl -lstdc++ -Wl,-rpath,/soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/kokkos/20210323-3.1/../../compiler/latest/linux/compiler/lib/intel64 -L/soft/restricted/CNDA/sdk/2021.04.30.001/oneapi/kokkos/20210323-3.1/../../compiler/latest/linux/compiler/lib/intel64 -liomp5 -lsycl -lOpenCL -lm -lpthread -ldl"
+# export EXTRA_LINK_FLAGS="-L$MKL_ROOT/lib/intel64 -lmkl_lapack95_lp64 -lmkl_intel_lp64 -lmkl_core -lmkl_intel_thread -lpthread -lm -lmkl_intel_ilp64  -lmkl_sycl -lstdc++ -Wl,-rpath,$MKL_ROOT/lib/intel64 -L$MKL_ROOT/lib/intel64 -liomp5 -lsycl -lOpenCL -lm -lpthread -ldl"
+export EXTRA_LINK_FLAGS=" -lmkl_lapack95_lp64 -lmkl_intel_lp64 -lmkl_core -lmkl_intel_thread -lpthread -lm -lmkl_intel_ilp64  -lmkl_sycl -lstdc++ -liomp5 -lsycl -lOpenCL -lm -lpthread -ldl"
```

Comment:
do we really need all these? cmake should pick up some of those, in particular the lapack ones

Reply:
but I don't want to delay this PR because of that side issue. so I approve